eslint.config.mjs

```diff
@@ -19,7 +19,8 @@ export default [
     'components/**/*Version.js',
     'components/**/apiExamples/**',
     'components/**/test/**',
-    'components/**/demo/**'
+    'components/**/demo/**',
+    'components/**/dist/**'
```

Comment:
Is there a reason you added the `dist` directory?

Reply:
eslint is working on bundler generated file. any reason why we want to lint the bundled file?

Reply:
<img width="802" alt="image" src="https://github.com/user-attachments/assets/19273a6b-1e83-4a33-b445-2644e2cfd4e2">

Reply:
You should remove the `dist` folder from the linters.

Reply:
Yes we can ignore these files and will mark as resolved.